bin/apisix

```diff
@@ -661,28 +661,32 @@ local function init_etcd(show_output)
     end
 
     local etcd_conf = yaml_conf.etcd
-    local uri = etcd_conf.host .. "/v2/keys" .. (etcd_conf.prefix or "")
-
     local timeout = etcd_conf.timeout or 3
+    local uri
+
+    for _, host in ipairs(yaml_conf.etcd.host) do
```

Comment:
this is wrong.

If the user specifies multiple etcd addresses, these etcd instances should belong to one cluster.

When we need to initialize etcd, as long as one of them is successfully initialized, it is considered successful.

Reply:
sure fixed

Reply:
that is the fake code, please take a view.

```lua
-- fake code
for i, host in ipairs(yaml_conf.etcd.host) do
    local is_success = true
    for _, dir_name in ipairs({"/routes", "/upstreams", ...}) do
        local ok = ...
        if not ok then
            is_success = false
            break
        end
    end

    if is_success then
        break
    end
end
```